NOTICE

```diff
@@ -15,6 +15,14 @@ cJSON
 Copyright (c) 2009-2017 Dave Gamble and cJSON contributors
 Licensed under the MIT License
 
+libyaml
```

Comment:
Strictly, these credits are not needed because you are not hosting libfyaml or asprintf.  (This is a good thing!)  However, there is no problem with acknowledging the usage.  So please modify this a little as follows:

This framework uses the libfyaml library (https://github.com/pantoniou/libfyaml), which is:
Copyright (c) 2019 Pantelis...
It also uses asprintf.c (https://github.com/jwerle/asprintf.c) which is:
Copyright (c) 2014...
Licensed under...